docs/index.rst

```diff
@@ -2436,6 +2387,49 @@ You can use the ``#[WithStory]`` attribute to load stories in your tests:
 
 If used on the class, the story will be loaded before each test method.
 
+Loading stories as fixtures in your database
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. versionadded:: 2.6
+
+    Command ``foundry:load-stories`` and attribute ``#[AsFixture]`` were added in 2.6.
+
+Using command ``bin/console foundry:load-stories``, you can load stories as fixtures in your database.
```

Comment:
Should we mention that if only a single story exists (the recipe default), no command arguments are required?